core/policyToken.go

```diff
@@ -75,17 +69,55 @@ type Metadata struct {
 	User string `json:"user"`
 }
 
-const (
-	vaultPolicyAPI      = "/v1/sys/policy/"
-	vaultDefaultPolicy  = "default"
-	vaultTokenTTL       = "1h"
-	vaultTokenCreateAPI = "/v1/auth/token/create"
-	vaultTokenDeleteAPI = "/v1/auth/token/delete"
-
-	tokenFileSuffix = "-token.json"
+// TokenID structure to serialize a token ID from its fs storage
+/*
+{
+	"request_id": "ded4b254-de08-8c23-fb95-3d9352dbe002",
+	"lease_id": "",
+	"renewable": false,
+	"lease_duration": 0,
+	"data": null,
+	"wrap_info": null,
+	"warnings": null,
+	"auth": {
+	  "client_token": "2f7b982d-dd49-41df-cca3-dbf23f166751",
+	  "accessor": "bbba3c5d-454b-3940-b20b-72d5a3e3cf3d",
+	  "policies": [
+		"admin",
+		"default"
+	  ],
+	  "metadata": null,
+	  "lease_duration": 3600,
+	  "renewable": true,
+	  "entity_id": ""
+	}
+  }
+*/
+/*
+type AutoGenerated struct {
+	RequestID     string      `json:"request_id"`
+	LeaseID       string      `json:"lease_id"`
+	Renewable     bool        `json:"renewable"`
+	LeaseDuration int         `json:"lease_duration"`
+	Data          interface{} `json:"data"`
+	WrapInfo      interface{} `json:"wrap_info"`
+	Warnings      interface{} `json:"warnings"`
+	Auth          struct {
+		ClientToken   string      `json:"client_token"`
+		Accessor      string      `json:"accessor"`
+		Policies      []string    `json:"policies"`
+		Metadata      interface{} `json:"metadata"`
+		LeaseDuration int         `json:"lease_duration"`
+		Renewable     bool        `json:"renewable"`
+		EntityID      string      `json:"entity_id"`
+	} `json:"auth"`
+}
 
-	contentType = "application/json"
-)
+// https://mholt.github.io/json-to-go/
```

Comment:
elaborating in a comment on how this site was used would be nice and save future readers from having to go to the site to see what it does and guess what it was used to generate